pkg/controller/kabaneroplatform/featured_collections.go

```diff
@@ -116,7 +116,9 @@ func reconcileFeaturedCollectionsV2(ctx context.Context, k *kabanerov1alpha1.Kab
 
 	ownerIsController := true
 	for _, c := range featured {
-		// For each collection, assure that a corresponding resource exists.
+		// For each collection, assure that a corresponding resource exists and it is at
+		// the highest level found among the repositories.
+		updateCollection := cl.Update
```

Comment:
Shouldn't the same update be needed for V1 collections (method above)? We support both V1 and V2.

Reply:
Yes but I'd like to abandon the V1 path.

Reply:
I opened #224 to propose removing the V1 collection path.